dali/operators/sequence/sequence_rearrange.cc

```diff
@@ -19,18 +19,20 @@
 namespace dali {
 
 DALI_SCHEMA(SequenceRearrange)
-    .DocStr(R"code(Rearrange the sequence stored as tensor.
-Assumes that the outermost dimension represents a sequence and other dimensions of input
-represent elements of that sequence. If layout is specified, the first dimension should
-be denoted as ``F`` indicating frames of the sequence.)code")
+    .DocStr(R"code(Rearranges the sequence that is stored as tensor.
+
+Assumes that the outermost dimension represents a sequence and the other dimensions of
+input represent elements of that sequence. If layout is specified, the first dimension
+should be denoted as ``F``, which indicates the frames of the sequence.)code")
     .NumInput(1)
     .NumOutput(1)
     .AllowSequences()
-    .AddArg("new_order", R"code(List describing new order for elements of each sample.
+    .AddArg("new_order", R"code(List that describes the new order for the elements in each sample.
+
 Output sequence at position ``i`` will contain element ``new_order[i]`` from input sequence.
```

Comment:
```suggestion
Output sequence at position ``i`` will contain the element at position ``new_order[i]`` in the input sequence.
```
Although I wonder - I think that pseudocode would be even easier to understand:
```suggestion
Output sequence at position ``i`` will contain element ``new_order[i]`` from input sequence::

  out[i, ...] = in[new_order[i], ...]
```

Reply:
Done